src/components/ArrowListItem/ArrowListItem.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { useFocus } from '../../hooks/useFocus';
+
+import './ArrowListItem.scss';
+import classNames from 'classnames';
+import { SoftKeyConsumer, SoftKeyManagerProps } from '../SoftKey/withSoftKeyManager';
+
+const prefixCls = 'kai-al';
+
+interface LocalProps {
+  primary: string;
+  secondary?: string;
+  focusClass?: string;
+  forwardedRef?: any;
+  index?: number;
+  onFocusChange?: (index: number) => void;
+  onClick?:() => void;
+}
+type Props = LocalProps 
+  & SoftKeyManagerProps;
+
+const ArrowListItem = React.memo<Props>(
+  props => {
+    const {
+      primary,
+      secondary,
+      focusClass,
+      forwardedRef,
+      index,
+      onFocusChange,
+      onClick,
+      softKeyManager
+    } = props;
+
+    const handleFocusChange = isNowFocused => {
+      if (isNowFocused) {
+        if(onFocusChange)
+          onFocusChange(index);
+        if(onClick)
+          softKeyManager.setCenterCallback(onClick);
+      }
```

Comment:
Use curly braces rather than just tabbing

Reply:
Done, added tslint to enforce this